tests/foreman/ui/test_organization.py

```diff
@@ -362,3 +362,26 @@ def test_positive_product_view_organization_switch(session, module_org, module_p
         assert session.product.search(module_product.name)
         session.organization.select(org_name="Default Organization")
         assert not session.product.search(module_product.name) == module_product.name
+
+
+def test_positive_prepare_for_sca_only_deprecation(target_sat):
+    """Verify that Simple Content Access endpoints are depreacated and will be required
+        for all organizations in Satellite 6.16
+
+    :id: 08539596-1bd3-4363-9737-e45f32ee5cbb
+
+    :expectedresults: Attepting to create an Organization with sca set to False, will throw
+        deprecation endpoint message
+    """
+    with target_sat.ui_session() as session:
+        session.organization.create(
+            {
+                'name': gen_string('alpha'),
+                'label': gen_string('alpha'),
+                'sca': False,
+            }
+        )
+        results = target_sat.execute('tail -100 /var/log/foreman/production.log').stdout
+    assert (
+        'Simple Content Access will be required for all organizations in Satellite 6.16' in results
+    )
```

Comment:
Isnt this warning logged in the notification panel ? And just being silently sent into logs only ?

If yes, please verify the notification mssg as well, If not should we force DEVs to log that in notification as well ?